messages/ignored_list.json

```diff
@@ -0,0 +1,8 @@
+{
+  "description": "check your local project package directories for forceignored files",
+  "examples": ["$ sfdx force:source:ignored", "$ sfdx force:source:ignored --sourcepath force-app"],
+  "flags": {
+    "sourcepath": "file or directory of files that the command checks for foreceignored files"
+  },
+  "invalidSourcePath": "File or directory '%s' doesn't exist in your project. Specify one that exists and rerun the command."
```

Comment:
is this message used?

Reply:
good call...not anymore.  SDR is gonna handle that